src/modules/organisation/domain/organisation.service.spec.ts

```diff
@@ -94,4 +95,34 @@ describe('OrganisationService', () => {
             });
         });
     });
+
+    describe('findAllOrganizations', () => {
```

Comment:
Test structure: when clause/scenario is missing.

Reply:
done